snippets/numpy/lib/type_check.py

```diff
@@ -0,0 +1,64 @@
+
+from ulab import numpy as np
+from ..core.multiarray import (asarray)
+from ..core.overrides import set_module
+
+@set_module('numpy')
+
+def _isreal(a):
```

Comment:
We can definitely add this function in C. https://numpy.org/doc/stable/reference/generated/numpy.isreal.html

Reply:
> @wired8 I have linked 4 issues here that would be closed by merging this. Are those correct?



> @wired8 I have linked 4 issues here that would be closed by merging this. Are those correct?

Yes, this closes all.